Editor/Utilities/DataObjTreeView.cs

```diff
@@ -14,6 +14,7 @@ internal class DataObjTreeView : TreeView
         TreeNode tree;
         TreeViewItem rootViewItem;
         List<TreeViewItem> rowList;
+        private int previousSelection = -1;
```

Comment:
1. I prefer to rename it to `previousSelectionId`.
2. When a node be open, it will select the root node. The `previousSelection` should be 1(the root id is 1, not 0).

Reply:
I changed the name, but as for the initial value being -1, I had set it to that, to act as a filter for the first initial click, but then is assigned to a proper value after a selection is made, as in some of my menus, upon loading them, I manually set the initial selection value to something other than 1, but I can certainly set it to 1 for this, I will just keep it like I have it in my own project.

Reply:
It's my mistake. 😥
The selection when node be opened will not trigger `NodeOnSelected`.
It should not be trigger `NodeOnDeselected` on the opened selection,  either.
You set the initial value to -1 is correct behaviour.
Please fallback to the prevous change.